libs/community/langchain_community/vectorstores/azure_cosmos_db.py

```diff
@@ -163,9 +177,12 @@ def delete_index(self) -> None:
 
     def create_index(
         self,
+        kind: str = "vector-ivf",
         num_lists: int = 100,
-        dimensions: int = 1536,
         similarity: CosmosDBSimilarityType = CosmosDBSimilarityType.COS,
+        dimensions: int = 1536,
+        m: int = 16,
+        ef_construction: int = 64,
```

Comment:
switching the order of these is a breaking change. can we keep (self, num_lists, dimensions, similarity) in that order and add all new args at the end?

Reply:
@baskaryan, made the changes as suggested, and fixed the linting issues as well.